R/steps-rcmdcheck.R

```diff
@@ -1,32 +1,33 @@
 RCMDcheck <- R6Class(
-  "RCMDcheck", inherit = TicStep,
+  "RCMDcheck",
+  inherit = TicStep,
 
   public = list(
-    initialize = function(warnings_are_errors = NULL, notes_are_errors = NULL,
-                          args = c("--no-manual", "--as-cran"),
-                          build_args = "--force", error_on = "warning",
-                          repos = repo_default(), timeout = Inf) {
+                initialize = function(warnings_are_errors = NULL, notes_are_errors = NULL,
```

Comment:
Can you please unindent?

Reply:
That was not on purpose. Playing around with vim ;)